### ISSUE TITLE
When I have parametrize in test, PT019 happens

### ISSUE THREAD

**Issue by Mulugruntz** [m-burst/flake8-pytest-style#173]
# Bug report

## What's wrong

In the following code:
```python
import pytest

parameters = [
    pytest.param(1, 2, 3, 3),
    pytest.param(2, 3, 4, 4),
    pytest.param(3, 4, 5, 5),
]


class TestClass:
    @pytest.mark.parametrize(("a", "b", "_", "_2"), parameters)
    @pytest.mark.usefixtures("_")
    @pytest.mark.usefixtures("_2")
    def test_one(self, a, b):
        assert a < b

    @pytest.mark.parametrize(("_", "_2", "c", "d"), parameters)
    def test_two(self, _, _2, c, d):
        assert c == d
```

I get the following warning:
```
test/unit/test_flake.py:17:5 PT019 fixture _ without value is injected as parameter, use @pytest.mark.usefixtures instead
test/unit/test_flake.py:17:5 PT019 fixture _2 without value is injected as parameter, use @pytest.mark.usefixtures instead
```
This happens when we have parametric tests and we don't want to consume all parameters.
We can see that using `usefixtures` silences the one from `test_one` successfully but should it be encouraged?
This is because the parameter was mistaken for a fixture.
To be noted that PyCharm is not happy about `test_one`, saying `Following arguments are not declared but provided by decorator: [_, _2] `. It's perfectly fine with `test_two`.

## How it should work

Parameters should be detected as parameters. And not as fixtures.
Flake8 should not request to add `@pytest.mark.usefixtures` (I think this is an undesirable piece of code here).
And `PT019` should not be triggered.

## System information

* Operating system: Fedora 35
* Python version: 3.7
```
pipenv run flake8 --version
3.9.2 (flake8-bugbear: 21.9.2, flake8-darglint: 1.8.1, flake8-pytest-style:
1.5.1, flake8-tidy-imports: 4.5.0, mccabe: 0.6.1, pycodestyle: 2.7.0,
pyflakes: 2.3.1) CPython 3.7.12 on Linux
```

**Comment by m-burst** [comment]
Hi @Mulugruntz,

My motivation behind PT019 was to avoid unused parameters in test functions.
This also applies to parameters generated via `parametrize`, so I do not consider this behaviour a bug.
However, I understand the rule is opinionated. You can always mute it with a `noqa` comment or disable the rule altogether in your configuration file.

**Comment by Mulugruntz** [comment]
Hi @m-burst 

I did mute it. However, objectively `PT019` says `fixture {name} without value ...` but **it is not** a fixture. So, we can see that it's objectively not "working as designed" and is therefore a bug.

tl;dr: PT019 refers to "name" as a fixture when "name" is not a fixture. Therefore it's a bug.
